.github/ISSUE_TEMPLATE/01_bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: 🐞 Bug report
+about: Something is broken
+---
+
+### Subject
+
+Describe the issue here.
+
+### Environment
+
+Describe your environment.
+At least, paste here the output of:
+
+```python
+from __future__ import print_function
+import platform
+import urllib3
+print("OS     : %s" % (platform.platform(), ))
+print("Python : %s" % (platform.python_build()[0], ))
+print("urllib3: %s" % (urllib3.__version__, ))
+```
+
+### Steps to reproduce
```

Comment:
All the headings should be title cased, Steps to Reproduce, Expected Behavior

Reply:
Done